projects/openmote-b/00std_spi/00std_spi.c

```diff
@@ -71,20 +71,25 @@ void spi_init() {
     SysCtrlPeripheralSleepEnable(SYS_CTRL_PERIPH_SSI1);
     SysCtrlPeripheralDeepSleepDisable(SYS_CTRL_PERIPH_SSI1);
 
-    SSIDisable(SSI1_BASE);
-    SSIClockSourceSet(SSI1_BASE, SSI_CLOCK_PIOSC);
+    SSIDisable(SSI1_BASE); // writes into SSI_CR1
+    SSIClockSourceSet(SSI1_BASE, SSI_CLOCK_PIOSC); // writes into SSI_CC
 
+    // configures IOC_Pxx_SEL register
     IOCPinConfigPeriphOutput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_CLK, IOC_MUX_OUT_SEL_SSI1_CLKOUT);
+    // configures IOC_Pxx_SEL register
     IOCPinConfigPeriphOutput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_TX, IOC_MUX_OUT_SEL_SSI1_TXD);
+    // configures hardware perihperal input selection (i.e. IOC_SSIRXD_SSI1)
     IOCPinConfigPeriphInput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_RX, IOC_SSIRXD_SSI1);
 
+    // set the pins as Hardware-controlled outputs (AFSEL register)
     GPIOPinTypeSSI(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_CLK );
     GPIOPinTypeSSI(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_RX );
```

Comment:
I'm not sure we want this pad to be hardware controlled.
What peripheral will control it? Is there a risk to it turns into an output?
I'd rather see it configured as a software input.